DFC.App.MatchSkills/Controllers/MatchDetailsController.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System.Collections.Generic;
+using DFC.App.MatchSkills.Models;
+using DFC.App.MatchSkills.ViewModels;
+using DFC.Personalisation.Domain.Models;
+using Microsoft.AspNetCore.DataProtection;
+using Microsoft.Extensions.Options;
+using Microsoft.AspNetCore.Mvc;
+
+namespace DFC.App.MatchSkills.Controllers
+{
+    public class MatchDetailsController : CompositeSessionController<MatchDetailsCompositeViewModel>
+    {
+        public MatchDetailsController(IDataProtectionProvider dataProtectionProvider, IOptions<CompositeSettings> compositeSettings) : base(dataProtectionProvider, compositeSettings)
+        {
+        }
+
+        [Route("/body/[controller]")]
```

Comment:
You don't need to put an attribute route on the override it's taken care of by the base class.